### PR TITLE
[F2PKEY-648] add deployment manifest for suppliers fabric-ca

### PR DESCRIPTION
**Pull request by azzahamdani**
add deployment manifest for suppliers fabric-ca
add deployment manifest for manifacturers fabric-ca
add deployment manifest for orderers fabric-ca
add deployment manifest for distributors fabric-ca

### CODE REVIEW

File: fabric-ca/suppliers/suppliers-ca-deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  template:
+    metadata:
+      labels:
+        app: suppliers-fabric-ca
+    spec:
+      containers:
+      - name: fabric-ca
+        image: hyperledger/fabric-ca:amd64-1.5
+
+        # command:
```
Comment (hinakoshibata) [inline fabric-ca/suppliers/suppliers-ca-deploy.yaml:21]: There is a code commented that should be deleted

**Review by hinakoshibata — CHANGES_REQUESTED** [review]
See comments

File: fabric-ca/suppliers/suppliers-ca-deploy.yaml (hunk 1 above)
Comment (azzahamdani) [reply fabric-ca/suppliers/suppliers-ca-deploy.yaml:21]: Allright

**Review by hinakoshibata — APPROVED** [review]
Code has been deleted 
Approved

**Pull request merged**